.goreleaser.yml

```diff
@@ -1,21 +1,16 @@
+# See http://goreleaser.com for valid values
 project_name: getmesh
 builds:
   - binary: getmesh
+    main: .
     env:
       - CGO_ENABLED=0
     goos:
       - linux
       - darwin
     goarch:
       - amd64
-archives:
-  - format: tar.gz
-    name_template: '{{ .Binary }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}'
-    files: [""] # necessary to exclude any files except the binary
-checksum:
-  name_template: 'checksums.txt'
-snapshot:
-  name_template: "{{ .Tag }}-next"
+      - arm64
```

Comment:
added this as it is no effort. Ack the downloads for istio won't likely exist, so I can remove it.